run_all_experiments.py

```diff
@@ -162,6 +198,19 @@ def parse_args() -> argparse.Namespace:
                       '--introspector-endpoint',
                       type=str,
                       default=introspector.DEFAULT_INTROSPECTOR_ENDPOINT)
+  parser.add_argument('-g',
+                      '--generate-benchmarks',
+                      help='Benchmarks to create',
```

Comment:
Can you better document this in the help string? 

This takes in a comma separated list of  heuristics / oracles? What are the allowed values?

Reply:
+1
If they are known functions from projects not yet integrated into `OSS-Fuzz,` would it be a good idea to provide them as a default value? If there are too many, we can read them from a file.

Reply:
Done, PTAL